api/migrations/20200526035639-auto_accept_and_rename_is_open.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn('calendar_events', 'is_requestable', {
+        allowNull: false,
+        type: Sequelize.BOOLEAN,
+        defaultValue: false,
+      }),
+      queryInterface.addColumn('calendar_events', 'auto_accept_request', {
+        allowNull: false,
+        type: Sequelize.BOOLEAN,
+        defaultValue: false,
+      }),
+
+      queryInterface.removeColumn('calendar_events', 'is_open'),
+      queryInterface.sequelize.query(
+        'ALTER TABLE calendar_events ADD CONSTRAINT auto_accept_condition CHECK (not(is_requestable IS FALSE AND auto_accept_request IS TRUE ));',
+      ),
+    ]);
```

Comment:
What if the 4th query gets performed before the first 2?

Chain the promises using `then` and callback.